libs/zzz/page-optimize/src/DiscConditionalCard.tsx

```
@@ -0,0 +1,160 @@
+        </CardContent>
+      )}
+    </CardThemed>
+  )
+}
+export function ConditionalToggle({
+  condMeta,
+}: {
+  condMeta: (typeof allDiscCondKeys)[DiscCondKey]
+}) {
+  const { database } = useDatabaseContext()
+  const character = useCharacterContext()!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)